### PR TITLE
Adds a systemd service file and the corresponding documentation

### PR DESCRIPTION
**Pull request by StuntsPT**
Following up from #1, I have made the PR so everyone can enjoy the systemd unit file.
I was considering doing things "right" by providing a JSON config file, but really, doing it "right" would be trying to get these patches upstream with OpenRGB. =-)
However, as a stopgap measure, it could be interesting to get a config parser on the main binary. As a python developer, I wouldn't even know where to start patching, though. Would this be something you would consider including? If yes, what would be a good place to start digging for a C config parser library? Or maybe this should get it's own separate issue?

Thanks,

Francisco

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,6 @@ The audio part of the headset works using the default ALSA USB soundcard drivers
 
 Right now I just wanted to track the very (VEEEERRRRYYY) basic app I wrote which allows you to set the LED color. I've also uploaded an example udev rule that sets the LEDs to purple whenever the headset is plugged in. Drop it in `/etc/udev/rules.d` if you want to try it out.
 
+The `sbx-h6-led.service` file is a systemd unit file that will set the headset's color to the factory default's  red (`ff0000`) and the brightness to `255` at startup. It will set the color to red (`ff0000`) and the brightness to `0` on shutdown to make sure the headset's LEDs are off when you turn off the machine. These values, can of course be changed to your liking. This was created mostly to deal with the fact that by default when you tunr your PC off, the LEDs on the headset will stay on. In order to use this service file, just copy it to `/etc/systemd/system/` and enable it with the command `systemctl enable sbx-h6-led.service` (root privileges required). Also note that the value `041e:3255` is 'hard-coded' in this systemd file. I assume this would be the same for all "SoundBlasterX H6" devices, but YMMV. You can verify if your headset has the same ID with the command `lsusb |grep "BlasterX"`.
```
Comment (Oscillope) [inline README.md]: Minor spelling error "turn your PC off"

File: sbx-h6-led.service [hunk 2]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=TurnSoundBlasterX-H6 LED ON at startup and OFF at Shutdown Service
```
Comment (Oscillope) [inline sbx-h6-led.service]: Suggest adding a space between `Turn` and `SoundBlaster`
```suggestion
Description=Turn SoundBlasterX-H6 LED ON at startup and OFF at Shutdown Service
```

**Review by Oscillope — APPROVED** [review]
Thanks for doing this! Just a few minor spelling comments, otherwise looks good!

I agree that the best thing to do would be to add SBX-H6 support to OpenRGB and deprecate this package instead of adding config parsing support to this code. I actually have patch in my fork of OpenRGB which adds support, though my fork is a few years old at this point. I'll try to take a look at updating my patch for the modern OpenRGB and submitting an upstream PR this weekend, now that I know that there's interest besides just my own :)

**Comment by StuntsPT** [comment]
>Thanks for doing this! Just a few minor spelling comments, otherwise looks good!
Thank you for your suggestions! I have updated the PR to fix them.

>I agree that the best thing to do would be to add SBX-H6 support to OpenRGB and deprecate this package instead of adding config parsing support to this code. I actually have patch in my fork of OpenRGB which adds support, though my fork is a few years old at this point. I'll try to take a look at updating my patch for the modern OpenRGB and submitting an upstream PR this weekend, now that I know that there's interest besides just my own :)
That would be super! Can you please point me towards your fork, so I can follow up?

The program in the meantime works as a great stop-gap measure. However, after using the systemd service for a few days on both my computers I found it to be nagging when the box is turned on without the headset plugged in. I will have to thing this through, but combining it with the `udev` rule you wrote should provide a better solution.

File: sbx-h6-led.service (hunk 2 above)
Comment (StuntsPT) [reply sbx-h6-led.service]: Solved via fc85857.

File: README.md (hunk 1 above)
Comment (StuntsPT) [reply README.md]: Solved via fc585857.

**Comment by StuntsPT** [comment]
I just wanted to follow up on this.
With your help, I was able to make a much more elegant [AUR package](https://aur.archlinux.org/packages/sbx-h6-rgb-git).
Instead of relying on just the systemd service file, or the udev rule, the package now relies on both, and makes them work together.
The systemd service file is not meant to be `enabled`, but the udev rule simply (re)starts it when the headphones are plugged in. The service file then takes over, and turns off the headphones on system shutdown. This way, there is no error message when the system is turned on without the headphones plugged in, and the rest of the functionality is maintained, at the cost of having 2 files instead of one.
Headphones color can be set on the systemd service file (defaults to red), and can even be changed with a restart.
An eventual next step would be allowing the color to be changed without root permissions (but that's for another day!).
Due to your accepting of my PR, my downstream AUR package does not bring any files with it, but rather "patches" the example udev rule file you provided to work with systemd.
Once again, thank you for your work and effort in building sbx-h6-rgb!

**Pull request merged**